smrf/framework/model_framework.py

```diff
@@ -43,35 +46,9 @@
 class SMRF:
     """
     SMRF - Spatial Modeling for Resources Framework
-
-    Attributes:
-        start_date: start_date read from configFile
-        end_date: end_date read from configFile
-        date_time: Numpy array of date_time objects between start_date and
-            end_date
-        config: Configuration file read in as dictionary
-        distribute: Dictionary the contains all the desired variables to
-            distribute and is initialized in
-            :func:`~smrf.framework.model_framework.create_distribution`
```

Comment:
Outdated and there are many more. Needs a dedicated documentation effort in general